scripts/release

```diff
@@ -1,30 +1,14 @@
 #!/usr/bin/env bash
-set -e
 
-# TODO this file could be gone
-RELEASED_LOG="/tmp/dotnet-pending-changes.md"
-THIS_VERSION=$(./scripts/bump --this-version)
+set -e
 
-# Generate the changelog with changes in this release
-./scripts/changelog --release-tag "$THIS_VERSION"
-git add CHANGELOG.md
-git commit -m "Update Changelog for Release $THIS_VERSION"
-git push origin master
+# Usage
+#
+# ./release
+#
 
 #publish
 dotnet pack -c Release
 dotnet build -c Release
 dotnet nuget push "$(pwd)"/Recurly/bin/Release/Recurly."$THIS_VERSION".nupkg -s https://api.nuget.org/v3/index.json -k "$NUGET_TOKEN"
```

Comment:
We'll need to use `RELEASE_TAG` here instead of `THIS_VERSION`. Same as well for [this branch](https://github.com/recurly/recurly-client-dotnet/pull/627/files#diff-738cff26e36d290aec109d76d56425fdf8ba8ac9c4aae6b8ecc65b79d484e6ecR13).